_posts/2023-06-28-Remilia-interview.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Interview wth contributor Remilia Scarlet"
+author: beta-zilliani,Remilia
+summary: "<b>Music playback projects in Crystal, games engines, and a bit of Lisp"
+---
+<https://remilia.sdf.org/>
+
+<https://www.youtube.com/watch?v=UX2in-whUik>
+
+_Conversation between Beta Ziliani and Remilia Scarlet_
+## A little about you. Who's Remilia?
+
+Well, I'm an open-source developer who has worked with quite a few languages over the years. I mainly do command line tools, but recently I've been interested in audio programming as well.
```

Comment:
Unless there's some very specific reason for it, I would suggest to drop such interjections. There's no need for filled pauses in written speach.
```suggestion
I'm an open-source developer who has worked with quite a few languages over the years. I mainly do command line tools, but recently I've been interested in audio programming as well.
```